File: snippets/fsharp/VS_Snippets_CLR_System/system.array.exists/fs/exists3.fs
module exists3

// <Snippet3>
open System

let  planets = 
    [| "Mercury"; "Venus"
       "Earth"; "Mars"; "Jupiter"
       "Saturn"; "Uranus"; "Neptune" |]

Array.Exists(planets, fun element -> element.StartsWith "M")
|> printfn "One or more planets begin with 'M': %O"

Array.Exists(planets, fun element -> element.StartsWith "T")
|> printfn "One or more planets begin with 'T': %O"

Array.Exists(planets, fun element -> element = "Pluto")
|> printfn "Is Pluto one of the planets? %O"

// The example displays the following output:
//       One or more planets begin with 'M': True
//       One or more planets begin with 'T': False
//       Is Pluto one of the planets? False
// </Snippet3>

File: snippets/csharp/System/Array/ExistsT/exists3.cs
// <Snippet3>
using System;

namespace Example
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] planets = { "Mercury", "Venus",
                "Earth", "Mars", "Jupiter",
                "Saturn", "Uranus", "Neptune" };

            Console.WriteLine($"One or more planets begin with 'M': {Array.Exists(planets, element => element.StartsWith("M"))}");

            Console.WriteLine($"One or more planets begin with 'T': {Array.Exists(planets, element => element.StartsWith("T"))}");

            Console.WriteLine($"Is Pluto one of the planets? {Array.Exists(planets, element => element == "Pluto")}");
        }
    }
}
// The example displays the following output:
//       One or more planets begin with 'M': True
//       One or more planets begin with 'T': False
//       Is Pluto one of the planets? False
// </Snippet3>
